server/sonar-web/src/main/js/helpers/urls.ts

```diff
@@ -48,6 +48,10 @@ export function getProjectUrl(key: string, branch?: string): Location {
   return { pathname: '/dashboard', query: { id: key, branch } };
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([typescript:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-1185%2Fhomepage_on_sonarqube&issues=AWHDQBb5ijWPeJ7YyCPn&open=AWHDQBb5ijWPeJ7YyCPn)

[ISSUE_KEY]: AWHDQBb5ijWPeJ7YyCPn